docker-compose.yml

```diff
@@ -2,67 +2,62 @@ version: "3"
 
 services:
 
-  gtt:
+  postgis:
+    image: postgis/postgis:14-master
+    environment:
+      POSTGRES_PASSWORD: ${DB_PASS}
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_DB: ${DB_NAME}
+    volumes:
+      - postgis:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "${DB_USER}", "-d", "${DB_NAME}"]
```

Comment:
Adding `"-h", "postgis"` may be better for stability.
Please see the following issue's comment.
https://github.com/postgis/docker-postgis/pull/297#issuecomment-1147603192